extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -376,10 +371,6 @@ private void applyBuildItems(Session session,
         containerImageBuildItem
                 .ifPresent(c -> session.resources().decorate(KNATIVE, new ApplyImageDecorator(knativeName, c.getImage())));
 
-        //Handle env variables
-        kubernetesEnvBuildItems.forEach(e -> session.resources()
```

Comment:
Interesting, so how can another extension add env variables to the k8s deployment without this snippet?

Reply:
This was just a left over from a previous iteration of the s2i implementation (as confirmed by @iocanel). No extensions need to do this anymore